### DISCUSSION TITLE
VERY Simple Clipface "logo" for the README

### DISCUSSION THREAD

**Discussion by ghost** [Hubro/clipface#60] [Ideas]
Hey Hubro! Here is a "logo" that you could toss into the top of the README if you'd like. I made it for the repository my friend and I were messing with, but I thought I'd share it with you as well in case you wanted something at the top of the README.

Here is the logo: https://snacks.dev/Clipface.png

EDIT: The font type is Montserrat so it's free to use FYI.

**Comment by Hubro** [comment]
Looks nice! If you made it yourself, could you export it as SVG? Since it's text based it's nicer to add to version control, and it's smaller, and it scales perfectly. Only advantages really :smile:

**Reply by ghost** [reply]
I made this in Photohop yesterday and I thought about making it an svg and just felt lazy about it, but I think that's a good idea because svg just makes so much more sense here.

Just went ahead and recreated it in Illustrator real quick. Does this work? https://snacks.dev/clipface_logo.svg
